### ISSUE TITLE
Getting Spammed with warnings "No token selected" every pan

### ISSUE THREAD

**Issue by insquared** [Garsondee/RTSControls#14]
A a GM I generally don't have a token selected 90% of the time, I get spammed with the warnings every 
time I pan, screenshot:
![Screen](https://github.com/Garsondee/RTSControls/assets/74970976/33d09d62-2bf9-4ba6-ba84-7f12e2578628)

**Comment by marioasr** [comment]
Even checking the option to hold shift, this warning keeps comming.

**Comment by roleoutgames** [comment]
Same issue, it is very distracting. Great module overall, but an option to toggle off the notification or simply build it to not trigger for GM's would be much appreciated.

**Comment by LonePaladin** [comment]
It's not just panning; _any_ right-click with no tokens selected is popping the error message. 1.2.4 or 1.2.5 must have introduced a new bug.

> 🎵 _Ninety-nine lines of bugs in the code_
> _Ninety-nine lines of bugs_
> _Pin one down_
> _Patch it around_
> . . .
> _One hundred twenty-seven lines of bugs in the code_

**Comment by Melkyor** [comment]
Hello,
Exactly the same for me.
- Foundry VTT 11 (build 315) + DND 5e 3.0.3

![Capture d'écran 2024-02-29 041523](https://github.com/Garsondee/RTSControls/assets/16270717/1db54a16-bb67-4f0f-82de-0740c74a553b)

- Unchecking the "Enable module" box changes nothing. The message “No token selected!” always appears with a right click
- Checking the "Shift right click to move" box does not change anything
- Disabling the module in the Foundry VTT Module Manager no longer displays the message. For now, this is the best solution; especially since as a DM, I use Right Click all the time to move the map. And if I leave the module active, I get the message every time I right click.

**Comment by fishrobe** [comment]
Yes same thing. ANY right click makes this pop up, even when hold shift is enabled.  (Foundry 11 v315, pf2e)

Me and my players love this module, but it is distracting enough that I am tempted to disable it until it is fixed.

**Comment by LagendaBergerak** [comment]
For this particular issue, i just comment the warning "no token..." inside the module.js, and it solve it for me,

**Comment by fishrobe** [comment]
> For this particular issue, i just comment the warning "no token..." inside the module.js, and it solve it for me,

Sorry, but can we pretend for a moment I have no idea how to do this?

**Comment by LagendaBergerak** [comment]
> > For this particular issue, i just comment the warning "no token..." inside the module.js, and it solve it for me,
> 
> Sorry, but can we pretend for a moment I have no idea how to do this?

Okay, first find where the mod is save, and find rtscontrol mod
![image](https://github.com/Garsondee/RTSControls/assets/97179042/07788bd5-3840-47a8-83b8-703e8bc75a8d)

inside rtscontrol folder, go to scripts folder, and open module.js
![image](https://github.com/Garsondee/RTSControls/assets/97179042/8d746b3b-ee25-4d15-b170-97471ada93b9)
![image](https://github.com/Garsondee/RTSControls/assets/97179042/e39c8ce4-70bd-4367-9584-35b889ca26c8)

then just find the line that have "no token selected", then just put // to command it
![image](https://github.com/Garsondee/RTSControls/assets/97179042/bc38c3e1-501c-4eea-b793-e6669243fdf6)

disclaimer: this is just temporary solution, and maybe the creator of the mod have his own reason on why he put that code, hope there will be a better way.

**Comment by fishrobe** [comment]
Thanks a load @Babysharkdad! Did the trick for now. (for anyone else looking who is using Forge VTT hosting, you will need to download RTS and edit it, and then upload it with the bazaar turned off).
